### PR TITLE
BCMOHAD-26159: HA Code and Initiative Code updates

### PR DESCRIPTION
**Pull request by npham49**
- Adding HA Code, PCN Community Code, Primary Care Network ID and Initiative Code fields.

### CODE REVIEW

File: src/main/java/ca/bc/gov/chefs/etl/forms/pcd/haHierarchy/model/Community.java [hunk 1]
```diff
@@ -12,6 +12,7 @@ public class Community implements IModel {
     private String communityName;
     private String hsiarServicePlanGapAnalysis;
     private String hsiarServicePlanGapAnalysisDate;
+    private String pcnCommunityCode;
```
Comment (weskubo-cgi) [inline src/main/java/ca/bc/gov/chefs/etl/forms/pcd/haHierarchy/model/Community.java:15]: I'm following up to see if it's too late to change the column name but I prefer communityCode.

File: src/main/java/ca/bc/gov/chefs/etl/constant/PCDConstants.java [hunk 2]
```diff
@@ -339,521 +314,225 @@ public class PCDConstants extends Constants {
...
-		HEADERS.put(HA_HIERARCHY_PRIMARY_CARE_INITIATIVES, new String[] {
-				"PRIMARY_CARE_NETWORK_ID", "PRIMARY_CARE_INITIATIVE_ID", "INITIATIVE_NAME", "INITIATIVE_TYPE",
-				"TYPE_OF_CARE"
-		});
-		HEADERS.put(HA_HIERARCHY_CLINIC, new String[] {
-				"PRIMARY_CARE_NETWORK_ID", "CLINIC_ID", "CLINIC_NAME", "CLINIC_TYPE"
-		});
+		HEADERS.put(HA_HIERARCHY_HEALTH_AUTHORITY, new String[] {"SUBMISSION_ID", "CREATED_AT", "LATE_ENTRY", "SUBMITTER_FULL_NAME", "SUBMITTER_USER_NAME", "SUBMITTER_EMAIL", "SUBMISSION_STATUS",
+				"SUBMISSION_VERSION", "SUBMISSION_FORM_NAME", "HEALTH_AUTHORITY", "HEALTH_AUTHORITY_CODE"});
+		HEADERS.put(HA_HIERARCHY_COMMUNITY, new String[] {"SUBMISSION_ID", "COMMUNITY_ID", "COMMUNITY_NAME", "HSIAR_SERVICE_PLAN_GAP_ANALYSIS", "HSIAR_SERV_PL_GAP_ANLYS_DATE", "PCN_COMMUNITY_CODE"});
```
Comment (weskubo-cgi) [inline src/main/java/ca/bc/gov/chefs/etl/constant/PCDConstants.java:333]: I'm following up to see if it's too late to change the column name but I prefer COMMUNITY_CODE.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]
Please see comments.

File: src/main/java/ca/bc/gov/chefs/etl/forms/pcd/haHierarchy/model/Community.java (hunk 1 above)
Comment (npham49) [reply src/main/java/ca/bc/gov/chefs/etl/forms/pcd/haHierarchy/model/Community.java]: This rename should be resolved in the latest commit!

File: src/main/java/ca/bc/gov/chefs/etl/constant/PCDConstants.java (hunk 2 above)
Comment (npham49) [reply src/main/java/ca/bc/gov/chefs/etl/constant/PCDConstants.java]: This rename should be resolved in the latest commit!

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**